### PR TITLE
feat(notifications): Fix ExternalActor Permissions

### PR DESCRIPTION
**Pull request by mgaeta**
We're unable to update ExternalActors in the Sentry organization. This PR removes the 2fa check for updating external actors.
@ceorourke @NisanthanNanthakumar

### CODE REVIEW

**Review by ceorourke — APPROVED** [review]
Approving to unblock us but I'd feel more comfortable if we had another set of eyes

**Comment by dcramer** [comment]
Can someone articulate the problem this is addressing for those of us not familiar with what these models are for?

**Comment by MeredithAnya** [comment]
> Can someone articulate the problem this is addressing for those of us not familiar with what these models are for?

Yeah I don't think I quite understand what the problem is, is this just when you are trying to update the external actor with a token?

**Pull request closed without merging**
